packages/datadog-plugin-bluebird/test/index.spec.js

```diff
@@ -3,3 +3,70 @@
 const assertPromise = require('../../dd-trace/test/plugins/promise')
 
 assertPromise('bluebird')
+
+assertPromise('bluebird', bluebird => {
+  // TODO: remove if statement when running tests only for versions ^2.11.0 and ^3.4.1
```

Comment:
added this filter as function thaat you can pass as an optional arg to `assertPromise`. Used `semver` here since it looks like this is what we use elsewhere, I think this ought to do the trick but lmk if i'm off based here